pyproject.toml

```diff
@@ -24,7 +24,7 @@ confluent-kafka = "^1.8.2"
 cachetools = "^5.0.0"
 redis = "^4.3.1"
 pynumaflow = "~0.2.6"
-numalogic = {version = "0.2.8", extras = ["mlflow"]}
+numalogic = {version = "~0.2.9", extras = ["mlflow"]}
```

Comment:
For non-numaproj dependencies, we use Caret requirements instead tilde requirements. Is there any reasons that we treat numaproj dependencies differently?

Reply:
The reason behind that is that numaproj dependencies have not reached a stable release yet (>= 1.0); so even patch updates can be breaking changes.